hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/ProtobufLogReader.java

```diff
@@ -446,7 +444,7 @@ protected boolean readNext(Entry entry) throws IOException {
         // Else restore our position to original location in hope that next time through we will
         // read successfully.
         LOG.debug("Encountered a malformed edit, seeking back to last good position in file, "
-          + "from {} to {}", inputStream.getPos(), originalPosition, eof);
+          + "from {} to {}", inputStream.getPos(), originalPosition);
```

Comment:
@Apache9 how about this change (line 447)? This change can be done if not the one above where we are throwing EOF (line 430-431)?